npm/linux-x64-gnu/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "good-fences-rs-linux-x64-gnu",
```

Comment:
I'd suggest deploying this under a namespace to avoid name sniping. I've reserved`@good-fences`, so maybe name these `@good-fences/api-*-*`?

Reply:
https://napi.rs/docs/introduction/getting-started#deep-dive
>Here it is recommended to distribute your package under [npm scope](https://docs.npmjs.com/creating-and-publishing-scoped-public-packages/) because @napi-rs/cli by default appends the different platform suffixes to the npm package name as the package name for the different platform binary distribution. Using npm scope will reduce the case of package name was taken.